src/System.Private.CoreLib/shared/System/Convert.cs

```diff
@@ -2148,17 +2149,17 @@ public static string ToString(DateTime value)
             return value.ToString();
         }
 
-        public static string ToString(DateTime value, IFormatProvider provider)
+        public static string ToString(DateTime value, IFormatProvider? provider)
         {
             return value.ToString(provider);
         }
 
-        public static string ToString(string value)
+        public static string? ToString(string? value) // TODO-NULLABLE: https://github.com/dotnet/roslyn/issues/26761
```

Comment:
Wondering what is the issue here (for TODO)

Reply:
To be able to annotate this method with an attribute that says the return value is null if and only if the input is null.